build.gradle

```diff
@@ -78,6 +78,12 @@ dependencies {
 	testCompile group: "junit", name: "junit", version: "4.12"
 }
 
+jar {
```

Comment:
hey @simonjhy  what does this do?

Reply:
This seems like it is copying too much into the jar.  The jar grew over 7 MBs with this change.  What is missing exactly that you were trying to include?

Reply:
ok

Reply:
commons-configuration.jar it seems intellij remove its support. We did not have this problem in last version intellij.